apiAuditLogs/apiAuditLogs.go

```diff
@@ -0,0 +1,277 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package apiAuditLogs
+
+import (
+	"fmt"
+	"go/ast"
+	"go/token"
+	"go/types"
+	"strings"
+
+	"github.com/mattermost/mattermost-govet/facts"
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+)
+
+var Analyzer = &analysis.Analyzer{
+	Name:     "apiAuditLogs",
+	Doc:      "check the audit logs usage in the API",
```

Comment:
check that audit records are properly created in the API layer.

Reply:
done